src/database/seeders/20190910075430-test-user.js

```diff
@@ -9,10 +9,22 @@ export function up(queryInterface, Sequelize) {
     username: 'johndoe',
     email: 'technitesdev@gmail.com',
     password: '123456',
+    company: 'Andela',
     is_verified: true,
     createdAt: new Date(),
     updatedAt: new Date()
-  }], {});
+  },
+  {
+    firstname: 'John',
+    lastname: 'Doe',
+    username: 'johndoe',
+    email: 'technitesdev2@gmail.com',
+    password: '123456',
+    is_verified: false,
```

Comment:
if you seed this user with verified false they won't be able to login thus getting the password will be a problem, and make sure email has a unque field

Reply:
That seed is for test purpose only, testing that non validated users can edit their profile